### PR TITLE
refactor(full aggregation scan): rework full aggregation scan according  to task https://github.com/scylladb/qa-tasks/issues/1004

### PR DESCRIPTION
**Pull request by temichus**
according to qa-task https://github.com/scylladb/qa-tasks/issues/1004 the following changes were made in full aggregation scan operations 1 remove randomizing from the aggregation scan
2 as a result - make more ERROR evens instead of WARNING
3 put a big timeout for the aggregation scan(30min by default) 
4 detect different failure reasons
5 update unit tests. coverage of sdcm/scan_operation_thread.py: 88%

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: longevity_test.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def _get_tombstone_gc_verification_params(self) -> dict:
         return params
 
     def _get_scan_operation_params(self) -> list[FullScanParams]:
-        params = self.params.get("run_fullscan", {})
+        params = self.params.get("run_fullscan", [])
```
Comment (fgelcer) [inline longevity_test.py:45]: i think it should not work, since #2823

File: unit_tests/test_scan_operation_thread.py [hunk 2]
```diff
@@ -151,8 +150,49 @@ def test_scan_negative_operation_timed_out(mode, severity, timeout, execute_mock
     assert "Severity.NORMAL" in all_events[0] and "period_type=begin" in all_events[0]
     assert f"Severity.{severity}" in all_events[1] and "period_type=end" in all_events[1]
 
+#######################################################################################################################
```
Comment (fgelcer) [inline unit_tests/test_scan_operation_thread.py:153]: IMHO we don't need this

File: longevity_test.py (hunk 1 above)
Comment (fruch) [reply longevity_test.py:45]: Guess which commit brought it back

https://github.com/scylladb/scylla-cluster-tests/blame/73dc21fe6b2fdd387986f9515bcdca9945002bc8/sdcm/sct_config.py#L1829

Comment (fgelcer) [reply longevity_test.py:45]: @temichus , please revert this change (from the commit above) and use as we currently do in the entire SCT

Comment (fgelcer) [reply longevity_test.py:45]: i always forget about this was removed from `self.params`

**Comment by fruch** [comment]
@temichus do you have references to jobs running with those changes ?

**Review by fruch — APPROVED** [review]

**Pull request merged**